evm/src/cpu/kernel/tests/account_code.rs

```diff
@@ -192,3 +195,182 @@ fn test_extcodecopy() -> Result<()> {
 
     Ok(())
 }
+
+/// Prepare the interpreter for storage tests by inserting all necessary accounts
+/// in the state trie, adding the code we want to context 1 and switching the context.
+fn prepare_interpreter_all_accounts(
+    interpreter: &mut Interpreter,
+    trie_inputs: TrieInputs,
+    to: [u8; 20],
```

Comment:
nit: `to` could be misleading (it's not necessarily the `to` address of a transaction).